mdx/guides/swap-tokens-with-0x-api.mdx

```diff
@@ -0,0 +1,278 @@
+All code snippets provided are designed to work in a browser environment with an injected web3 instance (like [Metamask](https://metamask.io/)). You can use the [npm web3 module](https://www.npmjs.com/package/web3) and modify these snippets to run them in a node environment.
+
+## Swap ETH for 1 DAI
+
+You can perform a two-line token swap like so: 
+
+```javascript
+(async () => {
+    // buying 1 DAI with ETH
+    const response = await fetch(
+        'https://api.0x.org/swap/v0/quote?buyToken=DAI&sellToken=ETH&buyAmount=1000000000000000000 (https://api.0x.org/swap/v0/quote?buyToken=DAI&sellToken=WETH&buyAmount=1000)'
```

Comment:
worth a tiny blurb explaining where `1000000000000000000` and `1000` come from. some may try to put in `1` and `1`